client/components/banner.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import style from '../bootstrap.min.css';
```

Comment:
this is nice, but you might not need to modularize bootstrap, since its loaded in our scaffold we can treat it as a global style set. but if u want to use a diff bootstrap then scaffold then perhaps this is needed.

Reply:
@cjm771 - I investigated that a bit, and it appears the build process only includes the css that's required by my components in the bundle, so it wouldn't be much code weight. Main reason for this is to implement css modules per requirements, and have the option to customize with local scope.